src/ble/BleLayer.cpp

```diff
@@ -600,7 +604,7 @@ bool BleLayer::HandleSubscribeReceived(BLE_CONNECTION_OBJECT connObj, const Chip
         return false;
     }
 
-    if (UUIDsMatch(&CHIP_BLE_CHAR_2_ID, charId))
+    if (UUIDsMatch(&CHIP_BLE_CHAR_2_ID, charId) || UUIDsMatch(&CHIP_BLE_CHAR_3_ID, charId))
```

Comment:
could we extract this comparison in a function?

I wonder what the compare checks for and why 2 and 3 are special.
Is this `IsKnownNonWriteCharacteristic(charID)` or `IsReadableDataCharacteristic(charId)` or something else?